hippunfold/config/snakebids.yml

```diff
@@ -420,6 +469,21 @@ nnunet_model:
   synthseg_v0.1: 'zenodo.org/record/8184230/files/trained_model.3d_fullres.Task102_synsegGenDetailed.nnUNetTrainerV2.model_best.tar'
   synthseg_v0.2: 'zenodo.org/record/8184230/files/trained_model.3d_fullres.Task203_synthseg.nnUNetTrainerV2.model_best.tar'
 
+atlas_links_osf:
```

Comment:
Do we need to have `_osf` in this key name? What about just calling them `atlas_urls` and `template_urls`? Then it is agnostic to where we keep things..

Reply:
good point, I'll make the change sometime early next week.

Reply:
done with ca786732530365c9a1ea394336ea6d1af665c5f0